README.md

```diff
@@ -23,6 +23,12 @@ Hide the browser instead of running headed at Cypress running
 
 Define what browser Cypress must run
 
+### `pr_author_username:
```

Comment:
Seria interessante atualizar em como consumir a action com esse novo input obrigatório, nas seções [Simple usage example](https://github.com/betrybe/cypress-evaluator-action#simple-usage-example) e [`How to get result output`](https://github.com/betrybe/cypress-evaluator-action#how-to-get-result-output), não?